common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1500,6 +1500,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVEQUERYNAME ("hive.query.name", null,
         "This named is used by Tez to set the dag name. This name in turn will appear on \n" +
         "the Tez UI representing the work that was done."),
+    HIVETEZJOBNAME("tez.job.name", null,
```

Comment:
The second value here, I believe is a 'default' value. The default should be `HIVE-%s`. Also please update comments to explain how this works.

Reply:
We can use 'set tez.job.name=<customized_job_name>;' or '--hiveconf ez.job.name=<customized_job_name>'  to use this function to customize Yarn application name when we run a sql